fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -454,7 +454,6 @@ public VisibleStateWaiter commitPreparedTransaction(long transactionId) throws U
                 // after state transform
                 transactionState.afterStateTransform(TransactionStatus.COMMITTED, txnOperated, callback, null);
             }
-            transactionState.prepareFinishChecker(db);
```

Comment:
is this safe, when will `checkCanFinish` be called in old publish mechnism?

Reply:
checkCanFinish only used in new publish